packages/atomic/src/pages/examples/ipx.html

```diff
@@ -63,12 +72,57 @@
         width: 5rem;
         height: 1.25rem;
       }
+
+      atomic-recs-list::part(label) {
+        font-size: var(--atomic-text-xl);
+        padding-top: 0.75rem;
+        padding-bottom: 0.25rem;
+      }
+
+      atomic-recs-list::part(outline) {
+        padding: 0.5rem;
+        border: none;
+      }
+
+      atomic-recs-list::part(result-list-grid-clickable) {
+        inset: unset;
+      }
+
+      atomic-recs-list::part(result-list-grid-content) {
+        width: 27rem;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
+
+      atomic-search-interface:not(.atomic-search-interface-search-executed)
+        > atomic-ipx-modal
+        > div[slot='body']
+        > atomic-layout-section {
+        display: none;
+      }
+
+      atomic-search-interface.atomic-search-interface-search-executed
```

Comment:
Should the `div[slot='body'] > atomic-layout-section` be unhidden at this stage?

Reply:
Yes we want to show the search results when the class `atomic-search-interface-search-executed` exists
and hide the list of recommendations.

Reply:
At-least while the initial search is being performed, we shouldn't hide the results, since we have placeholders.

Prior to the first search being executed at all, I think it should be the responsibility of the javascript that initializes the interface to hide it if they don't want it yet.

Reply:
At first when a user open a modal, we want to have recommandations (so no initial search).
When the user executes manually the initial search the placeholders are there (tested).

Is there a difference to have it in the JS or CSS if we want (in our case) to always hide the placeholders prior the inital search?